### PR TITLE
Enhancement #10

### PR DESCRIPTION
**Pull request by sourabhtk37**
Fix for #10

### CODE REVIEW

**Comment by sourabhtk37** [comment]
review plz @nishutosh

File: code_share/app_code_share/templates/app_code_share/homepage.html [hunk 1]
```diff
@@ -32,27 +35,102 @@
...
   </body>
-</html>
\ No newline at end of file
+
+  <script
+  src="https://code.jquery.com/jquery-3.2.1.min.js"
+  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+  crossorigin="anonymous"></script>
+    
+  <script>
```
Comment (sourabhtk37) [inline code_share/app_code_share/templates/app_code_share/homepage.html:76]: Script for disabling submit button @nishutosh

**Review by sourabhtk37 — COMMENTED** [review]
review plz @nishutosh

File: code_share/app_code_share/templates/app_code_share/homepage.html [hunk 2]
```diff
@@ -32,27 +35,102 @@
...
+    {% include "fork_github.html" %}
+
   </body>
-</html>
\ No newline at end of file
+
+  <script
+  src="https://code.jquery.com/jquery-3.2.1.min.js"
+  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+  crossorigin="anonymous"></script>
```
Comment (nishutosh) [inline code_share/app_code_share/templates/app_code_share/homepage.html:74]: remove jquery cdn link

File: code_share/app_code_share/templates/app_code_share/homepage.html (hunk 1 above)
Comment (nishutosh) [reply code_share/app_code_share/templates/app_code_share/homepage.html:76]: ya its looking fine

File: code_share/app_code_share/templates/app_code_share/homepage.html [hunk 3]
```diff
@@ -32,27 +35,102 @@
...
+    // 
+    // function check_content() {
+    //      if ($("#code_snippet").val() == "") {
+    //        alert("Cannot submit empty code snippet")
+    //        return false
+    //      }
+    //      else {
+    //        return true
+    //      }
+    //    }
```
Comment (nishutosh) [inline code_share/app_code_share/templates/app_code_share/homepage.html:139]: no it will annoy user ..users hate  alerts and all ...

Comment (sourabhtk37) [reply code_share/app_code_share/templates/app_code_share/homepage.html:139]: yeah agree

File: code_share/app_code_share/templates/app_code_share/homepage.html [hunk 4]
```diff
@@ -32,27 +35,102 @@
...
+      }
+
+    });
+
+
+
+    // Alert on form submission looked kinda ugly. But 
+    // feel free to suggest better error reporting.
+    // 
+    // function check_content() {
```
Comment (nishutosh) [inline code_share/app_code_share/templates/app_code_share/homepage.html]: no users hate alerts and all ..so leave it its fine by keeping buttons disabled

Comment (nishutosh) [inline code_share/app_code_share/templates/app_code_share/homepage.html]: k sry github error it just took some time to comment my previous one

File: code_share/app_code_share/templates/app_code_share/homepage.html (hunk 2 above)
Comment (sourabhtk37) [reply code_share/app_code_share/templates/app_code_share/homepage.html:74]: jQuery is required for event handler on submit button to work.
It does work on button without jQuery but not on Textarea, dunno why.

File: code_share/app_code_share/templates/app_code_share/homepage.html [hunk 5]
```diff
@@ -32,27 +35,102 @@
...
+  <script
+  src="https://code.jquery.com/jquery-3.2.1.min.js"
+  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+  crossorigin="anonymous"></script>
+    
+  <script>
+
+
+    // disables the submit button upon document ready
+    $(document).ready(function() {
```
Comment (nishutosh) [inline code_share/app_code_share/templates/app_code_share/homepage.html:80]: use window.onload=function(){}      why adding jquery use pure js

**Comment by sourabhtk37** [comment]
Merging this to development

**Pull request merged**